src/resources/post.ts

```diff
@@ -1,13 +1,50 @@
+import { AbstractInstanceType } from 'rest-hooks/lib/types';
 import BaseResource from './base';
 
+export interface Thumbnail {
+  readonly url: string;
+  readonly width: number;
+  readonly height: number;
+}
+
 export default class PostResource extends BaseResource {
-  static urlRoot = 'https://ymun-app.caseychow.org/_/items/posts';
+  // static urlRoot = 'https://ymun-app.caseychow.org/_/items/posts';
+  static urlRoot = 'https://ymun-app.caseychow.org/_/items/posts?fields=*.*&sort=-modified_on';
```

Comment:
See above about url roots not supporting parameters.

Reply:
JK you got the url solution working. That should be fine for our uses here. Note though that with the code you implemented we can’t add params in the useResource hook. Personally I still prefer to just make the two calls to useResource.